cmd/envoy/main.go

```diff
@@ -755,6 +741,28 @@ func daybreakRetire(c *cli.Context) (err error) {
 // Helper Functions
 //===========================================================================
 
+// Setup the context with the given actor ID (usually the function or command
+// name) and sets up the keychain to sign audit logs.
+func setupAuditLog(actorId string, ctx context.Context) (newCtx context.Context, err error) {
```

Comment:
Should the actorID be `"cli" or `"system"``? I think of actor as the thing that's initiating the action, e.g. the user ID of the logged in user - or is it the function name in other places as well?

Reply:
Ok, I do see there is an `ActorCLI` = so what is the ID?